# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

GraspCActionGoal action_goal
GraspCActionResult action_result
GraspCActionFeedback action_feedback
